src/polyfem/assembler/MacroStrain.hpp

```diff
@@ -0,0 +1,62 @@
+#pragma once
```

Comment:
why is this inside assembler?

Reply:
I consider this as some kind of "boundary condition", so I place it in the same folder as RHSAssembler